features/step_definitions/group_selection_steps.rb

```diff
@@ -419,6 +436,31 @@
   end
 end
 
+And(/.* should see eligibility failed error on dependent with resident role/) do
+  expect(find('.ineligible_ivl_row .ivl_errors')).to have_content("eligibility failed on active_individual_role")
+  expect(find(IvlChooseCoverage.dependent_1_checkbox).checked?).to be_falsey
+end
+
+Then(/.* should see all family members eligible$/) do
+  expect(find(IvlChooseCoverage.primary_checkbox).checked?).to be_truthy
+  expect(find(IvlChooseCoverage.dependent_1_checkbox).checked?).to be_truthy
+end
+
+And(/.* should see warning dialog on CoverAll selection/) do
+  expect(find("#WarningOnCoverAllSelection").visible?).to be_truthy
+
+  within('#WarningOnCoverAllSelection .modal-dialog') do
+    expect(page).to have_selector('.modal-title', text: 'Some people are eligible for DC Health Link')
```

Comment:
Is it possible to target something other than the text here (like an id or some other attribute)?  Or to use just the `site_key` helper which should be available in the Cucumbers to match the text?  We should avoid hardcoding English strings into our Cucumber expectations.

Reply:
replaced with ` Some people are eligible for #{site_short_name}`